docs/markdown/getting-started/installing-react-vis.md

```diff
@@ -0,0 +1,9 @@
+### Install the react-vis module
+
+If you want to use react-vis in your project, add it from the command line: 
+
+```
+npm install react-vis
+```
+
+(or yarn add react-vis - the following will assume that you use npm for concision's sake but you can substitute yarn if installed)
```

Comment:
maybe mention the stylesheet here? you talk about it being optional below, but im not sure thats true